### ISSUE TITLE
Investigate why tests related to tpf model fitting are hanging in AppVeyor

### ISSUE THREAD

**Issue by mirca** [KeplerGO/lightkurve#209]
See PR #208 for the unit test

**Comment by mirca** [comment]
I'm assigning this issue to myself, but this shouldn't stop anyone to investigating it .

**Comment by texadactyl** [comment]
I tried to reproduce this with an environment that I downloaded today.  See the attached files:

- test_tpfmodel.py  (slightly modified to work-around undefined PACKAGEDIR variable)
- appveyor.py  (main program I used to drive test_tpfmodel callable procedures)
- appveyor.log  (stdout + stderr)

I cannot hang any of the callable procedures but I did notice a consistent error reported by numpy while executing test_tpf_model_fitting():
```
/usr/lib/python3.6/importlib/_bootstrap.py:219: RuntimeWarning: numpy.dtype size changed, may indicate binary incompatibility. Expected 96, got 88
  return f(*args, **kwds)
```

[tests.zip](https://github.com/KeplerGO/lightkurve/files/2238177/tests.zip)

**Comment by texadactyl** [comment]
Ok, I downloaded the original github and left it as-is.  Then, I ran:
python3 setup.py test

No hangs.
84 passed, 33 skipped, 11 warnings in 34.35 seconds

[test.log](https://github.com/KeplerGO/lightkurve/files/2238231/test.log)

**Comment by barentsen** [comment]
A workaround was implemented in #208.
